app/models/form.rb

```diff
@@ -41,7 +44,7 @@ def has_live_version
   end
 
   def live_version
-    raise ActiveRecord::RecordNotFound if made_live_forms.blank?
+    raise ActiveRecord::RecordNotFound if made_live_forms.blank? || archived?
```

Comment:
may have to put in some other condition to check if runner or admin is trying to view last live version